programs/autocrat_v0/src/lib.rs

```diff
@@ -29,15 +29,20 @@ declare_id!("metaX99LHn3A7Gr7VAcCfXhpfocvpMpqQ3eyp3PGUUq");
 pub const SLOTS_PER_10_SECS: u64 = 25;
 pub const THREE_DAYS_IN_SLOTS: u64 = 3 * 24 * 60 * 6 * SLOTS_PER_10_SECS;
 
-// by default, the pass price needs to be 5% higher than the fail price
-pub const DEFAULT_PASS_THRESHOLD_BPS: u16 = 500;
+pub const TEN_DAYS_IN_SECONDS: i64 = 10 * 24 * 60 * 60;
+
+// by default, the pass price needs to be 3% higher than the fail price
+pub const DEFAULT_PASS_THRESHOLD_BPS: u16 = 300;
 
 // start at 10 SOL ($600 at current prices), decay by ~5 SOL per day
 pub const DEFAULT_BASE_BURN_LAMPORTS: u64 = 10 * solana_program::native_token::LAMPORTS_PER_SOL;
 pub const DEFAULT_BURN_DECAY_PER_SLOT_LAMPORTS: u64 = 23_150;
 
 pub const MAX_BPS: u16 = 10_000;
 
+// TWAP can only move by $5 per slot
+pub const MAX_OBSERVATION_CHANGE_PER_UPDATE_LOTS: u64 = 50_000;
```

Comment:
actually just realized this should be `5_000` since we've changed the base lot size, also would be good to add a test so we'd catch this next time

Reply:
Fixed.